Sources/MapboxMaps/Annotations/AnnotationOrchestratorImpl.swift

```diff
@@ -0,0 +1,262 @@
+    ///   - layerPosition: Optionally set the `LayerPosition` of the layer managed.
+    /// - Returns: An instance of `PolylineAnnotationManager`
+    internal func makePolylineAnnotationManager(id: String, layerPosition: LayerPosition?) -> PolylineAnnotationManager {
+        guard let displayLinkCoordinator = displayLinkCoordinator else {
+            fatalError("DisplayLinkCoordinator must be present when creating an annotation manager")
+        }
+        removeAnnotationManager(withId: id, warnIfRemoved: true, function: #function)
+        let annotationManager = PolylineAnnotationManager(
+            id: id,
+            style: style,
+            layerPosition: layerPosition,
+            displayLinkCoordinator: displayLinkCoordinator,
+            offsetLineStringCalculator: offsetLineStringCalculator)
+        annotationManagersByIdInternal[id] = annotationManager
+        return annotationManager
+    }
+
+    /// Creates a `CircleAnnotationManager` which is used to manage a collection of
+    /// `CircleAnnotation`s. Annotations persist across style changes. If an annotation manager with
+    /// the same `id` has already been created, the old one will be removed as if
+    /// `removeAnnotationManager(withId:)` had been called. `AnnotationOrchestrator`
+    ///  keeps a strong reference to any `CircleAnnotationManager` until it is removed.
+    /// - Parameters:
+    ///   - id: Optional string identifier for this manager.
+    ///   - layerPosition: Optionally set the `LayerPosition` of the layer managed.
+    /// - Returns: An instance of `CircleAnnotationManager`
+    internal func makeCircleAnnotationManager(id: String, layerPosition: LayerPosition?) -> CircleAnnotationManager {
+        guard let displayLinkCoordinator = displayLinkCoordinator else {
+            fatalError("DisplayLinkCoordinator must be present when creating an annotation manager")
+        }
+        removeAnnotationManager(withId: id, warnIfRemoved: true, function: #function)
+        let annotationManager = CircleAnnotationManager(
+            id: id,
+            style: style,
+            layerPosition: layerPosition,
+            displayLinkCoordinator: displayLinkCoordinator,
+            offsetPointCalculator: offsetPointCalculator)
+        annotationManagersByIdInternal[id] = annotationManager
+        return annotationManager
+    }
+
+    /// Removes an annotation manager, this will remove the underlying layer and source from the style.
+    /// A removed annotation manager will not be able to reuse anymore, you will need to create new annotation manger to add annotations.
+    /// - Parameter id: Identifer of annotation manager to remove
+    public func removeAnnotationManager(withId id: String) {
+        removeAnnotationManager(withId: id, warnIfRemoved: false, function: #function)
+    }
+
+    private func removeAnnotationManager(withId id: String, warnIfRemoved: Bool, function: StaticString) {
```

Comment:
Is it possible to make #function a default argument? So we can skip it in the function calls
```suggestion
    private func removeAnnotationManager(withId id: String, warnIfRemoved: Bool, function: StaticString = #function) {
```